storage/client_split_test.go

```diff
@@ -386,7 +386,7 @@ func TestStoreShouldSplit(t *testing.T) {
 
 	// Verify that the range is in fact split (give it a second for very slow test machines).
 	if err := util.IsTrueWithin(func() bool {
-		newRng := store.LookupRange(engine.KeyMax[:engine.KeyMaxLength-1], nil)
+		newRng := store.LookupRange(engine.KeyMax[:len(engine.KeyMax)-1], nil)
```

Comment:
`\xff` is an invalid encoding. We're not enforcing this now (and probably shouldn't at this level), but just in case it might be better to look up `\xff\x00`